modules/qna/src/backend/storage.ts

```diff
@@ -123,6 +125,7 @@ export default class Storage {
     const ids = await Promise.mapSeries(_.isArray(qna) ? qna : [qna], async (data, i) => {
       const id = makeID(data)
       await this.checkForDuplicatedQuestions(data, id)
+      data.lastModified = new Date()
       const item: QnaItem = { id, data }
```

Comment:
What do you think of that format instead ? 
`const item: QnaItem = { id, data: { ...data, lastModified: new Date() } }`